tox.ini

```diff
@@ -19,8 +19,6 @@ commands = coverage combine --rcfile {toxinidir}/.tox-coveragerc
 [testenv:packaging]
 changedir = {toxinidir}
 deps =
-   check-manifest==0.35
-   readme_renderer==17.2
```

Comment:
whoa whoa, [readme_renderer](https://pypi.org/project/readme_renderer/) says it supports Markdown, just maybe this version doesn't?

Reply:
Twine also use readme_renderer but only for .rst files and don't run any validation for markdown
https://github.com/pypa/twine/pull/421

Reply:
Welp, that settles it then!